tarsier/__main__.py

```diff
@@ -6,14 +6,21 @@
 from playwright.async_api import async_playwright
 
 from tarsier.core import Tarsier
-from tarsier.ocr import GoogleVisionOCRService
+from tarsier.ocr import GoogleVisionOCRService, MicrosoftAzureOCRService
 
 
-async def main(credentials_path: str, url: str, verbose: bool) -> None:
+
+async def main(credentials_path: str, ocr_service: str, url: str, verbose: bool) -> None:
```

Comment:
this will break existing versions, lets make ocr_service and optional dependancy:
```suggestion
async def main(credentials_path: str, url: str, verbose: bool, ocr_service: str = "google") -> None:
```

Reply:
Thanks for the feedback @awtkns, done.